README.md

```diff
@@ -15,29 +24,110 @@ The following endpoints are RESTful and allow full CRUD operations.
 - `/api/measurements`
 - `/api/sponsors`
 
+You can post a measurement to the api the following way (use the private api key of ApiConsumer):
+
+    curl -X POST 'http://localhost:3000/api/measurements' \
+        -H "Content-Type: application/json" \
+        -H "Authorization: Bearer 0123456789ABCDEF" \
+        -d '{"sensor_id": 1, "temperature": 20.7, "custom_attributes": {"foo": "bar"}}'
+
 ### Application Endpoints
 
 The following endpoints are optimized for mobile applications and return
 read-only aggregated data.
 
-- `/api/mobile_app/sensors`
-- `/api/mobile_app/sensors/<sensor-id>/sponsor`
-- `/api/mobile_app/sensors/<sensor-id>/daily_temperatures`
-- `/api/mobile_app/sensors/<sensor-id>/hourly_temperatures`
+**Sensors**
 
-### Authentication
+Endpoint: `/api/mobile_app/sensors`
 
-The API uses token based authentication. A sample request looks like this:
+JSON array element fields:
 
-    curl 'http://localhost:3000/api/measurements' -H "Authorization: Bearer 0123456789ABCDEF" -v
+| Field | Type | Description |
+| --- | --- | --- |
+| id | int | Sensor ID |
+| device_name | string | Sensor name |
+| caption | string | Caption describing the sensor |
+| created_at | uint | Unix timestamp (seconds) |
+| sponsor_id | uint? | Associated sponsor ID |
+| latitude | double? | Latitude (WGS84) |
+| longitude | double? | Longitude (WGS84) |
+| latest_temperature | double? | Latest temperature measurement |
```

Comment:
We're talking JSON. Those are all `number`.

Reply:
Good point.